R/derive_vars_last_dose.R

```diff
@@ -81,7 +81,7 @@
 #' @family deprecated
 #' @keywords deprecated
 #'
-#' @seealso [derive_var_last_dose_amt()], [derive_var_last_dose_date()],
```

Comment:
Think it makes more sense to get rid of them all or none, if you're to begin remove the seealso tags